src/api-doc/Entity Management API's.postman_collection.json

```diff
@@ -970,6 +970,188 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "targetedRoles",
```

Comment:
@MallanagoudaB Is this API called by app directly or is it via project service? If not then internal access token should not be required in this API

Reply:
we are not calling any service all the calls are internal